### PR TITLE
Adds new family friendly card game: MONO!

### PR DESCRIPTION
**Pull request by Gonzako**
## About the PR 
It adds a brand new card game for spessmen to enjoy on their spare time 
Sprites made by DisturbHerb

## Why's this needed? 
More miscellaneous content is always a great addition.

## Changelog

```changelog
(u)Gonzako, DisturbHerb
(+)Created a brand new, completely original card-game. Come play MONO!
```

### CODE REVIEW

**Comment by JackTrysGames** [comment]
_If this gets merged I swear I will grab a reverse card from the deck and throw it at people_

**Comment by DisturbHerb** [comment]
Think that this can just go in `playing_cards.dm` like all the other cards.

**Comment by NightmareChamillian** [comment]
I think you should make this a little more accessable, maybe put it into a crate or in crew lounge on a few maps.

**Comment by DisturbHerb** [comment]
> I think you should make this a little more accessable in order to encourage people to play it. Mybe put it into a crate or in crew lounge on a few maps.
> 
> > The clear nature of the file helps break up the "unaproachable monolith" some scripts become
> 
> Perhaps just add a commented out divider between it and the other games to break it up?

I feel that'd be out of the scope of this PR, maybe it'd be worth adding it to maps if this is merged.

File: code/obj/item/mono_card_game.dm [hunk 1]
```diff
@@ -0,0 +1,83 @@
+/obj/item/card_group/mono
+	card_style = "mono"
+	total_cards = 108
+	card_name = "MONO"
+	var/numToSpecial = list("10"="Block", "11"="Reverse", "12"="Draw 2")
```
Comment (pali6) [inline code/obj/item/mono_card_game.dm:5]:
```suggestion
	var/static/numToSpecial = list("10"="Block", "11"="Reverse", "12"="Draw 2")
```

File: code/obj/item/mono_card_game.dm [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+				setMonoCard(card, i, j)
+
+				var/obj/item/playing_card/card2 = new /obj/item/playing_card(src)
+				stored_cards += card2
+				setMonoCard(card2, i, j)
+			}
+		}
+		update_group_sprite()
+
+	proc/setMonoCard(var/obj/item/playing_card/target , var/suit, var/num)
```
Comment (pali6) [inline code/obj/item/mono_card_game.dm:39]: Why not make Mono cards a subtype of playing_card and make this the constructor?

**Review by pali6 — CHANGES_REQUESTED** [review]
No `;` at the end of lines. No `{ }` around control structures. Repalce `for(var/i = 1, i < 5, i++)` with `for(var/i in 1 to 4)`.

File: code/obj/item/mono_card_game.dm (hunk 2 above)
Comment (Gonzako) [reply code/obj/item/mono_card_game.dm:28]: Feedback pushed!

File: code/obj/item/mono_card_game.dm [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+			else  //If I fuck up generation this will get people complaining
+				target.name = "Wild Draw 4"
+				num = 1
+
+
+		target.icon_state = "[card_style]-[suit]-[num]"
+		update_card_information(target)
+		target.update_stored_info()
+		*/
+
```
Comment (pali6) [inline code/obj/item/mono_card_game.dm:61]:
```suggestion
```

**Review by pali6 — APPROVED** [review]

**Comment by pali6** [comment]
The .txt file should probably be renamed to .html but eh whatever.

**Pull request merged**